docs/components/organization/organization-list.mdx

```diff
@@ -6,10 +6,10 @@ description: Clerk's <OrganizationList /> component is used to display organizat
 # `<OrganizationList />` component
 
 <Images
-  alt="Organization List Example"
   width={496}
   height={564}
-  src="/docs/images/ui-components/component-org_list.svg"
+  src="/docs/images/ui-components/organization-list.svg"
```

Comment:
Broken for me ([preview](https://docs-preview-936.clerkpreview.com/docs/components/organization/organization-list))
![image](https://github.com/clerk/clerk-docs/assets/5156915/65311606-fe10-4399-ac27-f8601c8da25c)